src/views/ManageList.jsx

```diff
@@ -1,7 +1,90 @@
-export function ManageList() {
+import { useState } from 'react';
+import { addItem } from '../api/firebase';
+
+export function ManageList({ listPath }) {
+	const [itemName, setItemName] = useState('');
+	const [daysUntilNextPurchase, setDaysUntilNextPurchase] = useState(0);
+
+	const handleChangeItem = (e) => {
+		setItemName(e.target.value);
+	};
+
+	const handleChangeDate = (e) => {
+		setDaysUntilNextPurchase(parseInt(e.target.value));
+	};
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+		try {
+			const newItem = await addItem(listPath, {
+				itemName,
+				daysUntilNextPurchase,
+			});
+			alert('Item saved successfully.');
+		} catch (error) {
+			alert('Item not saved successfully.');
+		}
+		setItemName('');
+		document.getElementById('item-form').reset();
+	};
+
 	return (
-		<p>
-			Hello from the <code>/manage-list</code> page!
-		</p>
+		<div>
+			<form id="item-form" onSubmit={handleSubmit}>
+				<div>
+					<label htmlFor="item">Item Name</label>
+					<input
+						value={itemName}
+						onChange={handleChangeItem}
+						name="itemName"
+						type="text"
+						id="item"
+						required
+					/>
+				</div>
+
+				<div>
+					<label htmlFor="daysUntilNextPurchase">
+						How soon do you think you'll purchase this item again?
+					</label>
+					<ul>
+						<li>
+							<label htmlFor="soon"> Soon (7 days)</label>
+							<input
+								value={7}
```

Comment:
Consider making this your default value, to enhance accessibility. This can be achieved by adding another attribute here. If you chose to do this, you'll also need to update the initial state value to be "7".

Reply:
We initially leave this part of the form unselected to account for accidental form submissions where an item name would be provided but the first radio button would already be selected, before the user has a chance to select it on their own. Whereas if there is no default radio button value, the user would see an error message, asking them to select an option first. See below screenshot.

![image](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/78065152/d966946d-7abc-4ccd-b272-dd63af4eaa3c)

Reply:
@trushmi What do you think?